src/custompresets.asm

```diff
@@ -0,0 +1,460 @@
+; --------------
+; Custom Presets
+; --------------
+
+print pc, " custom presets start"
+
+PresetSlot: ; 1AEh size slots
+    dw $0000, $01AE, $035C, $050A, $06B8, $0866, $0A14, $0BC2
+    dw $0D70, $0F1E, $10CC, $127A, $1428, $15D6, $1784, $1932
+    dw $1AE0, $1C8E, $1E3C, $1FEA, $2198, $2346, $24F4, $26A2
+    dw $2850, $29FE, $2BAC, $2D5A, $2F08, $30B6, $3264, $3412
+    dw $35C0, $376E, $391C, $3ACA, $3C78, $3E26, $3FD4, $4182
+    dw $4330, $44DE, $468C, $483A, $49E8, $4B96, $4D44
+
+custom_preset_save:
+{
+    LDA !sram_custom_preset_slot : ASL : TAX
```

Comment:
If we allocated 512 bytes or 200h of space for each preset, that would allow us to add a bit more stuff in the future without forcing a compatibility break.  It might also be easier to reuse this space in the future, in case someone wants 20 custom presets and 20... whatever the next cool feature is.

We could also get rid of the PresetSlot table since we could just ASL : XBA : TAX to multiply !sram_custom_preset_slot by 200h.